docs/cloudformation_compatibility.rst

```diff
@@ -0,0 +1,97 @@
+CloudFormation Compatibility Section
+===============
+
+.. contents::
+
+SAM is built ontop of CloudFormation Transforms. Therefore, we need to support different CloudFormation Capabilities like: Attributes, Intrinsic functions, etc.
+
+CloudFormation Resources Attributes
+
+======================== ========================
+     Attribute Name             Supported?
+======================== ========================
+CreationPolicy           Not Currently
+DeletionPolicy           Not Currently
+DependsOn                `DependsOn Attribute`_
+Metadata                 Not Currently
+UpdatePolicy             Not Currently
+======================== ========================
+
+.. _DependsOn Attribute:
+
+DependsOn Attribute:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: yaml
+
+  LambdaFunction:
+    DependsOn: SomeOtherResources
+    Type: AWS::Serverless::Function
+    ...
+
+CloudFormation Intrinsic Funtions
+-------------------
+Currently, we do not support all Intrinsic Functions for all Property Values. Please see below tables for a details on which Intrinsic Functions can be used on a given field.
+
+The Condition Function is not currently supported on any ``AWS::Serverless::*`` Resource type
+
+.. code:: yaml
+
+  LambdaFunction:
+    Condition: SomeCondition
+    Type: AWS::Serverless::Function
+    ...
+
+
+AWS::Serverless::Function
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+======================== ======================== ========================
+     Property Name        Intrinsic(s) Supported          Reaons
+======================== ======================== ========================
+Handler                          Ref Only
```

Comment:
Not true. Handler, Runtime, Description, MemorySize, Timeout - We support all intrinsics.

Reply:
I thought I tested this, but must have been doing something else. Will update to reflect what is actually supported